econml/utilities.py

```diff
@@ -38,6 +38,24 @@ def transform(self, X):
         return X
 
 
+def parse_final_model_params(coef, intercept, d_y, d_t, d_t_in, bias_part_of_coef, fit_cate_intercept):
+    dt = d_t
+    if (d_t_in != d_t) and (d_t[0]==1):  # binary treatment
+        dt = ()
+    cate_intercept = None
+    if bias_part_of_coef:
+        cate_coef = coef.reshape(d_y + dt + (-1,))[..., 1:]
+        if fit_cate_intercept:
+            cate_intercept = coef.reshape(d_y + dt + (-1,))[..., 0]
+    else:
+        cate_coef = coef.reshape(d_y + dt + (-1,))
+        if fit_cate_intercept:
+            cate_intercept = np.reshape(intercept, d_y + dt)
+    if (cate_intercept is not None) and (np.ndim(cate_intercept) == 0):
```

Comment:
I don't understand this part - if `np.ndim` is 0, isn't the intercept already a scalar?

Reply:
No it ends up being e.g. np.array(.5). Behaves like a scalar but has this weird array around it.